test.py

```diff
@@ -563,16 +563,65 @@ def run_test(framework, cluster, repo_root):
         logger.info(msg, runtests_script, framework.name)
         raise CommandFailure(cmd_args)
 
+def setup_junit_xml():
+    """Try to download and import junit_xml"""
```

Comment:
It's possible to make this a container dependency, but I don't know how to update the container, and we don't have the ability to quickly update the container.  In short, I don't believe that's likely to be a better outcome.